apps/nextjs-website/src/components/molecules/RelatedResources/RelatedResources.tsx

```diff
@@ -0,0 +1,23 @@
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+import { Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { GuideCard, GuideCardProps } from '../GuideCard/GuideCard';
+
+interface RelatedResourcesProps {
+  guide: GuideCardProps;
```

Comment:
The component should take an array of GuideCardProps, not a single element and I think you should change the property name to `resources`